src/test/java/edu/cornell/kfs/fp/document/service/impl/CuDisbursementVoucherDefaultDueDateServiceImplTest.java

```diff
@@ -1,9 +1,13 @@
 package edu.cornell.kfs.fp.document.service.impl;
 
+import static org.mockito.ArgumentMatchers.any;
 import static org.junit.Assert.*;
+import static org.mockito.Mockito.when;
```

Comment:
Other areas of this class are already using `Mockito.when()` instead of the static import variant. You may want to adjust this test so that the import can be removed.